parsl/executors/taskvine/executor.py

```diff
@@ -88,10 +71,11 @@ class TaskVineExecutor(BlockProviderExecutor, putils.RepresentationMixin):
             with respect to other executors.
             Default is "TaskVineExecutor".
 
-        use_factory: bool
-            Choose to whether use either the Parsl provider or
-            TaskVine factory to scale workers.
-            Default is False.
+        worker_launch_method: str
+            Choose to use Parsl provider, TaskVine factory, or
+            manual user-provided workers to scale workers.
+            Options are among {'provider', 'factory', 'manual'}.
```

Comment:
theres a type annotation that's stricter than `str` for this kind of list of strings, that would look something like:

`Union[Literal['provider'], Literal['factory'], Literal['manual']]`

which I think would mean `typeguard` would detect invalid values in this parameter.